### PR TITLE
Add new make target: fixtures/rpm-erratum

### PR DESCRIPTION
**Pull request by Ichimonji10**
The new make target generates a JSON erratum file that can be imported
into a Pulp RPM repository.

### CODE REVIEW

**Comment by Ichimonji10** [comment]
Required by https://github.com/PulpQE/pulp-smash/pull/310

**Comment by elyezer** [comment]
Thank you

File: rpm/gen-erratum.sh [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+# Given a path to an RPM package, echo the package epoch.
+#
+# e.g. assets/walrus-5.21-1.noarch.rpm → 0
+get_rpm_epoch() {
+    local filename="$(basename "${1}")"
+    # If an epoch is listed, return it. Otherwise, assume an epoch of 0.
+    case "${filename}" in
+        *'!'*)
+            echo "${parts[0]#\!*}"
```
Comment (Ichimonji10) [inline rpm/gen-erratum.sh]: Whoops! This should be a reference to `filename`, not `parts`.

Comment (elyezer) [inline rpm/gen-erratum.sh]: Whoops! That was hiding very well when I read the code. Thanks for fixing it.

**Comment by elyezer** [comment]
LGTM

**Pull request merged**